spec/support/factories.rb

```diff
@@ -15,6 +15,18 @@
     author_id '1'
   end
 
+  factory :dynamic_book, class: Book do
+    sequence(:title) do |i|
+      i = i.to_s.rjust(2, "0") if i < 10
```

Comment:
So basically, the leading zero is just for proper ordering of the books for test pagination.